dev-python/shiboken/shiboken-9999.ebuild

```diff
@@ -1,44 +1,59 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
 
-PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
 
-inherit cmake-utils python-r1 git-r3
+inherit cmake-utils llvm python-r1 git-r3
 
-DESCRIPTION="A tool for creating Python bindings for C++ libraries"
-HOMEPAGE="https://wiki.qt.io/Pyside"
+DESCRIPTION="Tool for creating Python bindings for C++ libraries"
+HOMEPAGE="https://wiki.qt.io/PySide2"
 EGIT_REPO_URI=(
-	"git://code.qt.io/pyside/${PN}.git"
-	"https://code.qt.io/git/pyside/${PN}.git"
+	"git://code.qt.io/pyside/shiboken.git"
+	"https://code.qt.io/git/pyside/shiboken.git"
 )
 
 LICENSE="LGPL-2.1"
-SLOT="0"
+SLOT="2"
 KEYWORDS=""
 IUSE="test"
-
 REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
+# Minimum Qt version required, shiboken2 requires the qAsConst() function first
+# introduced by Qt 5.7.
+#
+# For safety, shiboken2 is automatically rebuilt on all Qt bumps. Even minor Qt
+# bumps could break backward compatibility or introduce new functionality.
+# Moreover, shiboken2 appears to violate Qt's privacy encapsulation.
+QT_PV="5.7.1:5="
+
+#FIXME: Update the "clang" dependency to require some minimum and possibly
+#maximum range of CLang versions. Due to insufficient upstream documentation and
+#ongoing development, this range is currently unknown.
 RDEPEND="
 	${PYTHON_DEPS}
-	dev-libs/libxml2
-	dev-libs/libxslt
-	dev-qt/qtcore:5
-	dev-qt/qtxml:5
-	dev-qt/qtxmlpatterns:5
+	sys-devel/clang:*
```

Comment:
Interesting. What about the `shiboken` binary and the `/usr/lib64/libshiboken2*` libraries?